src/ocm/components/clusterConfiguration/DiscoveryImageForm.tsx

```diff
@@ -47,24 +52,29 @@ const DiscoveryImageForm = ({
     formikActions: FormikHelpers<DiscoveryImageFormValues>,
   ) => {
     if (cluster.id && infraEnv?.id) {
-      try {
-        const { updatedCluster } = await DiscoveryImageFormService.update(
-          cluster.id,
-          cluster.tags,
-          infraEnv.id,
-          formValues,
-        );
-        await onSuccess();
-        dispatch(updateCluster(updatedCluster));
-      } catch (error) {
-        handleApiError(error, () => {
-          formikActions.setStatus({
-            error: {
-              title: 'Failed to download the discovery Image',
-              message: getApiErrorMessage(error),
-            },
+      if (formValues['imageType'] !== 'full-iso' && formValues['imageType'] !== 'minimal-iso') {
```

Comment:
I'm curious about this too, @ammont82, this condition seems unnecessarily complicated.

Reply:
ImageType only have two values 'full-iso' and 'minimal-iso' because when you select ipxe the image type not changes in the cluster. In the form we add the option 'discovery-image-ipxe' but then in DiscoveryImageFormValues we use the type ImageCreateParams that have the type ImageType inside.

Reply:
I think I see what's going on... When you put 
```suggestion
      if (formValues['imageType'] == 'discovery-image-ipxe') {
```
it gives you errors, right? That is because `formValues` is of type `DiscoveryImageFormValues` which is defined [here](https://github.com/openshift-assisted/assisted-ui-lib/blob/e0403af7397bcce17df95c8ce8a77f234147b9fc/src/common/components/clusterConfiguration/DiscoveryImageConfigForm.tsx#L51) as

```
export type DiscoveryImageFormValues = ImageCreateParams & ProxyFieldsType;
```
and indeed doesn't allow `'discovery-image-ipxe` as an imageType value. You will need to define a new type that does allow for this value -- we are setting it anyway -- and use that type for `formValues`.

Reply:
Maybe I can change in the other PR that I have opened. We can merge this PR in order to finish the PR about trusted certificates.

Reply:
If it's something smaller that can be fixed now, I think it's best to add it here and to deliver this feature the best as possible.

Reply:
Changes done